client/js/transports/websocket.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: "WebSocketTransport"
+description: "A lightweight transport for WebSocket based connections with Pipecat"
+---
+
+`WebSocketTransport` enables a purely WebSocket based connection between clients and your Pipecat application. It implements bidirectional audio and video streaming using a WebSocket for real-time communication.
+
+This transport is intended for lightweight implementations, particularly for local development and testing. It expects your Pipecat server to include the corresponding [`WebSocketTransport` server-side](/server/services/transport/websocket-server) implementation.
+
+<Warning>
+The `WebSocketTransport` is best suited for server-server applications and prototyping client/server apps.
+
+For client/server production applications, we strongly recommend using a WebRTC-based transport for robust network and media handling. For more on WebRTC vs. Websocket communication, check out [this article](https://voiceaiandvoiceagents.com/#websockets-webrtc).
+</Warning>
+
+## Usage
+
+### Basic Setup
+
+```javascript
+import { PipecatClient } from "@pipecat-ai/client-js";
+import {
+  WebSocketTransport,
+  TwilioSerializer,
+} from "@pipecat-ai/websocket-transport";
+
+const pcClient = new PipecatClient({
+  transport: new WebSocketTransport ({
+    serializer: new TwilioSerializer(),
+    recorderSampleRate: 8000,
+    playerSampleRate: 8000,
+  }),
+  enableCam: false, // Default camera off
+  enableMic: true, // Default microphone on
+  callbacks: {
+    // Event handlers
+  },
+});
+
+await pcClient.connect({
+    endpoint: 'https://your-server/connect',
+});
+```
+
+## API Reference
+
+### Constructor Options
+
+```typescript
+type WebSocketTransportOptions = {
+  ws_url?: string;
+  serializer?: WebSocketSerializer;
+  recorderSampleRate?: number;
+  playerSampleRate?: number;
+};
+```
+
+#### Properties
+
+<ParamField name="ws_url" type="string">
+  URL of the WebSocket server. This is the endpoint your client will connect to for WebSocket communication.
+</ParamField>
+
+<ParamField name="serializer" type="WebSocketSerializer" default="ProtobufFrameSerializer">
+  The serializer to use for encoding/decoding messages sent over the WebSocket connection. The websocket-transport package provides two serializer options:
+  - `ProtobufFrameSerializer`: Uses Protocol Buffers for serialization.
+  - `TwilioSerializer`: Uses Twilio's serialization format.
```

Comment:
This might be a good place to highlight what I mentioned above:
- The main purpose of the TwilioSerializer is to allow testing the bots we build to work with Twilio without making phone calls.

Reply:
done